packages/protobuf-example/package.json

```diff
@@ -7,10 +7,11 @@
     "build": "../../node_modules/typescript/bin/tsc --project tsconfig.json --module ES2015 --outDir ./dist/esm --declaration --declarationDir ./dist/types",
     "add-person": "node dist/esm/add-person.js addressbook.bin",
     "list-people": "node dist/esm/list-people.js addressbook.bin",
-    "buf:generate": "buf generate"
+    "generate": "buf generate"
```

Comment:
as written, this would still use a `buf` command in a path (and not from the NPM), wouldn't it?

Reply:
Same as with any other npm script, this runs with a modified $PATH, looking at `node_modules/.bin` first.

Reply:
ohhh right right, gotcha, thanks.